smart_meter_texas/const.py

```diff
@@ -1,23 +1,28 @@
 import datetime
 
-BASE_HOSTNAME = "smartmetertexas.com"
-BASE_URL = "https://" + BASE_HOSTNAME + "/"
-BASE_ENDPOINT = BASE_URL + "api"
-AUTH_ENDPOINT = "/user/authenticate"
-DASHBOARD_ENDPOINT = "/dashboard"
-LATEST_OD_READ_ENDPOINT = "/usage/latestodrread"
-METER_ENDPOINT = "/meter"
-OD_READ_ENDPOINT = "/ondemandread"
-INTERVAL_SYNCH = "/adhoc/intervalsynch"
+BASE_HOSTNAME = "www.smartmetertexas.com"
+BASE_URL = "https://" + BASE_HOSTNAME
+BASE_ENDPOINT = BASE_URL 
+AUTH_ENDPOINT = "/commonapi/user/authenticate"
+LATEST_OD_READ_ENDPOINT = "/api/usage/latestodrread"
+METER_ENDPOINT = "/api/meter"
+OD_READ_ENDPOINT = "/api/ondemandread"
+INTERVAL_SYNCH = "/api/adhoc/intervalsynch"
 
 USER_AGENT_TEMPLATE = (
     "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_6) "
     "AppleWebKit/537.36 (KHTML, like Gecko) "
     "Chrome/85.0.{BUILD}.{REV} Safari/537.36"
 )
 CLIENT_HEADERS = {
-    "Content-Type": "application/json",
-    "Accept": "application/json",
+    "accept": "application/json, text/plain, */*",
+    "accept-language": "en-US,en;q=0.9",
+    "cache-control": "no-cache",
+    "content-type": "application/json",
+    "origin": "https://www.smartmetertexas.com",
+    "pragma": "no-cache",
+    "referer": "https://www.smartmetertexas.com/home",
+    "user-agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36",
```

Comment:
Shouldn't we be using the `USER_AGENT_TEMPLATE` here?

Reply:
yes, it was redundant

Reply:
Done. I had to update the chrome version